multimodal-search-pdf/README.md

```diff
@@ -0,0 +1,129 @@
+<a href="https://twitter.com/intent/tweet?text=%F0%9F%91%8DCheck+out+Jina%3A+the+New+Open-Source+Solution+for+Neural+Information+Retrieval+%F0%9F%94%8D%40JinaAI_&url=https%3A%2F%2Fgithub.com%2Fjina-ai%2Fjina&hashtags=JinaSearch&original_referer=http%3A%2F%2Fgithub.com%2F&tw_p=tweetbutton" target="_blank">
+  <img src="https://github.com/jina-ai/jina/blob/master/.github/badges/twitter-badge.svg"
+       alt="tweet button" title="👍Share Jina with your friends on Twitter"></img>
+</a>
+[![Python 3.7 3.8](https://github.com/jina-ai/jina/blob/master/.github/badges/python-badge.svg "Jina supports Python 3.7 and above")](#)
+[![Docker](https://github.com/jina-ai/jina/blob/master/.github/badges/docker-badge.svg "Jina is multi-arch ready, can run on differnt architectures")](https://hub.docker.com/r/jinaai/jina/tags)
+
+</p>
+
+This is an example of using [Jina](http://www.jina.ai)'s neural search framework to search text, image and PDF files in indexed PDF data.
+
+In this example, we will index the PDF files in a multimodal way. That is, we extract and index both text and images from PDF files.
+
+What's included in this example:
+
+- Search text, image, pdf (better partial pdf) all in one flow or in separate flows
+- Leverage Jina Recursive Document Representation for to segment and encode text
+- Parallel peas to reduce indexing time
+- Customized executors to better fit the needs
+- Detailed docstrings for YAML files to help you understand Jina App
+
+- [PDF search](#pdf-search)
+  * [Use toy data](#use-toy-data)
+  * [Install](#install)
+  * [Run](#run)
+  * [Start the Server](#start-the-server)
+  * [Query via REST API](#query-via-rest-api)
+  * [Documentation](#documentation)
+  * [Community](#community)
+  * [License](#license)
+    
+
+## Use toy data
+
+We have included several PDF blogs as toy data in [`toy_data`](toy_data). This data is ready to use with this example.
+You can also use other PDF files supported by [`pdfplumber`](https://github.com/jsvine/pdfplumber).
+
+## Install
+
+```bash
+pip install -r requirements.txt
+```
+
+## Run
+
+| Command | Description |
+| :--- | :--- |
+| ``python app.py -t index`` | To index files/data |
+| ``python app.py -t query`` | To run query on the index |
+| ``python app.py -t dryrun`` | Sanity check on the topology |
+
+## Start the Server
+
+``` bash
+python app.py -t query_restful
+```
+
+## Query via REST API
+
+When the REST gateway is enabled, Jina uses the [data URI scheme](https://en.wikipedia.org/wiki/Data_URI_scheme) to represent multimedia data. Simply organize your picture(s) into this scheme and send a POST request to `http://0.0.0.0:45670/api/search`, e.g.:
+
+``` bash
+curl --request POST -d '{"top_k": 10, "mode": "search",  "data": ["jina hello multimodal"]}' -H 'Content-Type: application/json' 'http://0.0.0.0:45670/api/search'
```

Comment:
I expect users to ask us how to query other data, maybe provide an example for how to include an image in the data field? 

Could you also provide a sample result? Just the JSON output returned so user know what they expect to see back?

Reply:
Yes, this is a good suggestion! I will add the results. But it's not so easy to do curl for images or files because it's too long. 😂

Reply:
But the results are quite messy, not sure if it's ok to put here.

Reply:
For files and images: can you expand sections of Markdown? I think we have feature in our readme. 

I would do it despite the bad results.

Reply:
yes